test/test_init.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+from opensquirrel import Circuit, CircuitBuilder
+from opensquirrel.ir import Init
+
+
+@pytest.mark.parametrize(
+    ("cqasm_string", "expected_result"),
+    [
+        ("version 3.0; qubit[2] q; init q[1]; init q[0]", "version 3.0\n\nqubit[2] q\n\ninit q[1]\ninit q[0]\n"),
+        (
+            "version 3.0; qubit[4] q; init q[2:3]; init q[1, 0]",
+            "version 3.0\n\nqubit[4] q\n\ninit q[2]\ninit q[3]\ninit q[1]\ninit q[0]\n",
+        ),
```

Comment:
In this case, I really think two lines help.
And would you prefer to write the output program in several lines, I leave that up to you as well.

```suggestion
        (
            "version 3.0; qubit[2] q; init q[1]; init q[0]",
            "version 3.0\n\nqubit[2] q\n\ninit q[1]\ninit q[0]\n"
        ),
        (
            "version 3.0; qubit[4] q; init q[2:3]; init q[1, 0]",
            "version 3.0\n\nqubit[4] q\n\ninit q[2]\ninit q[3]\ninit q[1]\ninit q[0]\n",
        ),
```

Reply:
I accepted your changes, but `ruff format` prescribes the old form...

Reply:
Maybe worth using `# fmt: skip`?